docs/dev-env-first-time-contributors.md

```diff
@@ -294,7 +294,23 @@ Welcome to Ubuntu 14.04.1 LTS (GNU/Linux 4.4.0-21-generic x86_64)
 Congrats, you're now inside the Zulip dev environment!
 
 You can confirm this by looking at the command prompt, which starts with
-`(zulip-venv)`.
+`(zulip-venv)`. In case it does not, your development environment might be haulted
+while provisioning the vagrant guest. Check the `/var/log/zulip_provision.log`
+file present inside the guest machine to find it ending with something similar
+to the following lines in case of successful completion:
+
+```
+....
+
+Zulip development environment setup succeeded!
+Get:1 http://archive.ubuntu.com trusty InRelease
+Ign http://archive.ubuntu.com trusty InRelease
+....
+```
+If you encounter an incomplete /var/log/zulip_provision.log file, you need
```

Comment:
It would be better to surround the filepath and name with backticks to format them in a fixed-width font, for consistency with other places we do this in our documentation.